### PR TITLE
Fix css for icons

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

**Comment by pfanguin** [comment]
Test link : https://mf-geoadmin3.dev.bgdi.ch/dev_css_icons

**Comment by oterral** [comment]
Can't you do it these modifications in the font awesome files ?

**Comment by cedricmoullet** [comment]
@pfanguin Could you please answer oli's question ?

**Comment by pfanguin** [comment]
The best solution that I have found

**Comment by oterral** [comment]
I think, adding a font-size different for each icon is really bad,  we loose the purpose of using font-awesome.  We already do that in special case and it's not good.

@pfanguin Can you really not modify  the SVG to avoid using the `font-size` rules?

**Comment by pfanguin** [comment]
Better for you ?

**Comment by pfanguin** [comment]
jenkins test

**Comment by pfanguin** [comment]
jenkins test

File: src/style/app.less [hunk 1]
```diff
@@ -749,8 +756,11 @@ a .ga-icon-separator {
 
   .icon-reorder, .icon-remove {
     display: none;
+    vertical-align: middle;
```
Comment (oterral) [inline src/style/app.less]: icon-remove don't need the vertical-align

**Comment by oterral** [comment]
There is something that still bother me. A font-aweosme icon must be aligned with a text by default (default css is vertical-align: baseline) here it's not the case. 

Are you able to provide an HTML test page with all the icons we have when you build the font?. Like this one http://fortawesome.github.io/Font-Awesome/3.2.1/cheatsheet/

With this kind of page we could easily see if icons are well-aligned and well-sized easily.

**Comment by oterral** [comment]
jenkins test

**Pull request merged**